src/components/value-renderers/Title.tsx

```diff
@@ -1,21 +1,16 @@
 /** @jsxImportSource @emotion/react */
```

Comment:
This can be removed once you only use styled components in a file.

Reply:
This job will be done at the end oh the refactor